specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingAccount.json

```diff
@@ -1,1029 +1,670 @@
             "schema": {
-              "$ref": "#/definitions/BillingAccountUpdateRequest"
+              "$ref": "#/definitions/BillingAccountPatch"
             }
           }
         ],
         "responses": {
           "200": {
-            "description": "OK. The request has succeeded.",
+            "description": "A billing account.",
             "schema": {
               "$ref": "#/definitions/BillingAccount"
             }
           },
           "202": {
-            "description": "Accepted. Billing account update is in progress.",
+            "description": "An operation accepted result that returns a 'Location' header that the client can poll to view the result of the operation.",
             "headers": {
               "Location": {
                 "type": "string"
               }
             }
           },
           "default": {
-            "description": "Error response describing why the operation failed.",
+            "description": "Default response",
             "schema": {
-              "$ref": "./types.json#/definitions/ArmError"
+              "$ref": "./types.json#/definitions/ArmErrorResponse"
             }
           }
         },
-        "tags": [
-          "BillingAccounts"
-        ],
-        "x-ms-examples": {
-          "billingAccountsPatch": {
-            "$ref": "./examples/billingAccountsPatch.json"
-          }
-        },
         "x-ms-long-running-operation": true,
         "x-ms-long-running-operation-options": {
-          "final-state-via": "azure-async-operation"
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "BillingAccountsUpdate": {
+            "$ref": "./examples/billingAccountsUpdate.json"
+          },
+          "BillingAccountUpdateWithPONumber": {
+            "$ref": "./examples/billingAccountUpdateWithPONumber.json"
+          }
         }
       }
     },
-    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/listInvoiceSectionsWithCreateSubscriptionPermission": {
-      "post": {
-        "description": "Lists the invoice sections for which the user has permission to create Azure subscriptions. The operation is supported only for billing accounts with agreement type Microsoft Customer Agreement.",
+    "/providers/Microsoft.Billing/billingAccounts": {
+      "get": {
+        "tags": [
+          "BillingAccount"
+        ],
+        "description": "Lists the billing accounts that a user has access to.",
         "externalDocs": {
           "url": "https://docs.microsoft.com/en-us/rest/api/billing/"
         },
-        "operationId": "BillingAccounts_ListInvoiceSectionsByCreateSubscriptionPermission",
+        "operationId": "BillingAccounts_List",
+        "produces": [
+          "application/json"
+        ],
         "parameters": [
           {
-            "$ref": "./types.json#/parameters/apiVersionParameter"
+            "in": "query",
+            "name": "includeAllBillingAccounts",
```

Comment:
Should this be "includeAll" to be consistent with "includeDeleted"? I.e. "include<adjective>" without the noun (resource type).